internal/commands/app/diff.go

```diff
@@ -53,26 +55,49 @@ func (cmd *CommandDiff) Flags() []flags.Flag {
 		flags.StringFlag{
 			Value: &cmd.inputs.RemoteApp,
 			Meta: flags.Meta{
-				Name: "remote",
+				Name: flags.FlagRemote,
 				Usage: flags.Usage{
 					Description: "Specify the name or ID of a Realm app to diff",
 				},
 			},
 		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludeNodeModules,
+			Meta: flags.Meta{
+				Name: flags.FlagIncludeNodeModules,
+				Usage: flags.Usage{
+					Description: "Include Realm app dependencies in the diff from an archive file",
+					Note:        "The allowed formats are as a directory or compressed into a .zip, .tar, .tar.gz, or .tgz file",
+				},
+			},
+		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludePackageJSON,
+			Meta: flags.Meta{
+				Name: flags.FlagIncludePackageJSON,
+				Usage: flags.Usage{
+					Description: "Include Realm app dependencies in the diff from a package.json file",
+					Note:        "--include modules will be ignored if presented with this flag, node_modules.zip will be fetched if no package.json was configured",
```

Comment:
you'll notice this pattern exists in `commands/logs` package
```suggestion
					Note:        `"--include-node-modules" will be ignored if presented with this flag, node_modules.zip will be fetched if no package.json was configured`,
```

---

but also I thought we were erroring out now if the two flags were both set?  if so, we should probably just remove this `Note`

Reply:
yeah will just delete